cubic-server/LivingEntity.cpp

```diff
@@ -0,0 +1,42 @@
+#include "LivingEntity.hpp"
+#include "Player.hpp"
+
+/*
+ * @brief Attack the entity
+ *
+ * @param damage The damage to deal
+ * @param source The source of the damage
+ */
+void LivingEntity::attack(float damage, Vector3<double> source)
+{
+    std::vector<Player *> players =  _dim->getPlayerList();
+
+    // compute knockback
+    double knockback = 1000;
+    Vector3<double> direction = (source - _pos) * knockback;
```

Comment:
I think it would be better to dissociate damage and knockback into 2 functions.
ex: the void inflict damage but no knockback